db/version_set.cc

```diff
@@ -756,6 +756,13 @@ Version::~Version() {
       }
     }
   }
+
+  // Schedule background deletion of VersionStorageInfo (which includes Arena)
+  VersionStorageInfo* storage_to_delete = storage_info_;
+  env_->Schedule([](void* arg) {
```

Comment:
We know this part of the code is costly, but it’s not clear where that maps to in the flame graph, since the labels are truncated and only show leading characters. Could you help identify and highlight that section?